docs/configuration.rst

```diff
@@ -0,0 +1,26 @@
+Configuration Reference
+=======================
+
+``towncrier`` has many knobs and switches you can use, to customise it to your project's needs.
+The setup in the `Quick Start <quickstart.html>`_ doesn't touch on many, but this document will detail each of these options for you!
+
+For how to perform common customisation tasks, see `Customisation <customisation/index.html>`_.
+
+``[tool.towncrier]``
+--------------------
+
+All configuration for ``towncrier`` sits inside ``pyproject.toml``, under the ``tool.towncrier`` namespace.
+Please see `the TOML GitHub repo <https://github.com/toml-lang/toml>`_ for how to write TOML.
+
+Top level keys
+~~~~~~~~~~~~~~
+
+- ``package`` -- The package name of your project. (Python projects only)
+- ``package_dir`` -- The folder your package lives. ``./`` by default, some projects might need to use ``src``. (Python projects only)
+- ``newsfile`` -- The filename of your news file. ``NEWS.rst`` by default.
+- ``fragment_directory`` -- If you are not storing your newsfragments in your Python package, or aren't using Python, this is the path to where your newsfragments will be put.
```

Comment:
* What about "the path to where your newsfragments will be put – defaults to XXX"
* "will be put" by whom? by towncrier? Maybe better "will be collected" or "where towncrier will search for newsfragments"?
* Buh, this is a *long* line.

Reply:
This option actually is named just `directory`.